### PR TITLE
Experimental mobile client support

### PR DESCRIPTION
**Pull request by qtmdev**
Modifications to provide experimental support for mobile clients

### CODE REVIEW

File: u2f-gae-demo/war/js/u2fdemo.js [hunk 1]
```diff
@@ -202,3 +202,38 @@ function onError(code, enrolling) {
...
+    var reqId = ++u2f.reqCounter_;
+    u2f.callbackMap_[reqId] = callback;
+    var req = {
+      type: u2f.MessageTypes.U2F_SIGN_REQUEST,
+      signRequests: signRequests,
+      timeoutSeconds: (typeof opt_timeoutSeconds !== 'undefined' ?
+          opt_timeoutSeconds : u2f.EXTENSION_TIMEOUT_SEC),
+      requestId: reqId
+    };
+    var str = JSON.stringify(req);
```
Comment (leshi) [inline u2f-gae-demo/war/js/u2fdemo.js]: Shouldn't:

    var str = JSON.stringify(req);
    var url = "u2f://auth?" + encodeURI(str);
    location.replace(url);

be a function?

**Comment by qtmdev** [comment]
The redirections have been condensed to a single function.

File: u2f-gae-demo/war/js/u2fdemo.js [hunk 2]
```diff
@@ -202,3 +202,40 @@ function onError(code, enrolling) {
...
+
+  u2f.callbackMap_ = {};
+  u2f.sign = function(signRequests, callback, opt_timeoutSeconds) {
+    var reqId = ++u2f.reqCounter_;
+    u2f.callbackMap_[reqId] = callback;
+    var req = {
+      type: u2f.MessageTypes.U2F_SIGN_REQUEST,
+      signRequests: signRequests,
+      timeoutSeconds: (typeof opt_timeoutSeconds !== 'undefined' ?
+          opt_timeoutSeconds : u2f.EXTENSION_TIMEOUT_SEC),
```
Comment (leshi) [inline u2f-gae-demo/war/js/u2fdemo.js]: weird spacing here?

**Comment by leshi** [comment]
LGTM

**Pull request merged**